packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.controller.js

```diff
@@ -0,0 +1,19 @@
+import { FIELD_NAME_REGEX } from './add-device.constants';
+
+export default class SoftphoneAddDeviceController {
+  /* @ngInject */
+  constructor(coreURLBuilder, $stateParams) {
+    this.linkGenerated = false;
+    this.model = { name: null, recordLink: '-' };
```

Comment:
```suggestion
    this.model = { name: null };
```
Shouldn't in your model because it isn't some thing that you send in api.